gdpr/anonymizers/model_anonymizers.py

```diff
@@ -342,29 +342,33 @@ def update_obj(self, obj: Model, legal_reason: Optional[LegalReason] = None,
         else:
             raw_local_fields = [i for i in parsed_fields.local_fields if
                                 self.is_field_anonymized(obj, i) and self[i].get_is_reversible(obj)]
-        update_dict = {name: self.get_value_from_obj(self[name], obj, name, anonymization) for name in raw_local_fields}
-
-        if self.anonymize_reversion(obj):
-            from reversion.models import Version
-            from gdpr.utils import get_reversion_local_field_dict
-            versions: List[Version] = self.get_reversion_versions(obj)
-            versions_update_dict = [
-                (
-                    version,
-                    {
-                        name: self.get_value_from_version(self[name], obj, version, name, anonymization=anonymization)
-                        for name in raw_local_fields
-                        if name in get_reversion_local_field_dict(version)
-                    }
+
+        if raw_local_fields:
```

Comment:
if there is nothing to anonymize no action is necessary.. For example all reversion data was get from DB but for nothing